docs/coordinate.md

```diff
@@ -0,0 +1,44 @@
+
+`Coordinate` parses `Latitude` and `Longitude`
+
+You can use the type `Coordinate` data type for storing coordinates.
+Coordinate can be defined by this format `<Latitude>,<Longitude>`: `41.40338, 2.17403`.
+The first number in your latitude coordinate is between -90 and 90.
+The first number in your longitude coordinate is between -180 and 180.
+
+```py
+from pydantic import BaseModel
+
+from pydantic_extra_types.coordinate import Longitude, Latitude, Coordinate
+
+
+class Lat(BaseModel):
+    lat: Latitude
+
+
+class Long(BaseModel):
+    long: Longitude
```

Comment:
I know this is simple and I am not sure if you agree with it, but I would rename the `Lat/lat` and `Long/long` in the examples because of the `long` type in Python.

Reply:
Maybe `Latit/Longit` ?

Reply:
Lat/Lng are common forms.

See https://developers.google.com/maps/documentation/javascript/examples/map-latlng-literal and https://pub.dev/documentation/latlong2/0.8.2/latlong2/LatLng-class.html

Reply:
Thanks, I've changed to `Lng`